app-benchmarks/stress-ng/stress_ng-0.13.05.recipe

```diff
@@ -38,25 +38,31 @@ SOURCE_URI="https://github.com/ColinIanKing/stress-ng/archive/V$portVersion.zip"
 SOURCE_DIR="stress-ng-$portVersion"
 CHECKSUM_SHA256="442c07327c5534566670a85dc67d50b1c643250aa09cc7129c793705d294cbeb"
 
+if [ $effectiveTargetArchitecture = x86 ]; then
+	PATCHES="stress_ng_x86-$portVersion.patchset"
+fi
+
 ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
-	stress_ng = $portVersion
-	cmd:stress_ng = $portVersion
+	stress_ng$secondaryArchSuffix = $portVersion
+	cmd:stress_ng$secondaryArchSuffix = $portVersion
 	"
 REQUIRES="
-	haiku
-	gcc_syslibs
-	lib:libz
+	haiku$secondaryArchSuffix
+	gcc${secondaryArchSuffix}_syslibs
```

Comment:
I think this one can be removed

Reply:
gcc_syslibs is required because of libatomic, otherwise we get a warning during package build. (error if using strict mode)

gcc_syslibs_devel can be removed from BUILD_REQUIRES.

Reply:
then declare the library itself: lib:libatomic

Reply:
ok that makes sense